client/src/pages/Login/Login.jsx

```diff
@@ -0,0 +1,241 @@
+                setUsername(value);
+              }}
+              localStorageKey="email-login"
+            />
+            <TextInput
+              inputType={'password'}
+              placeholder={'Password'}
+              onChange={(value) => {
+                setPassword(value);
+              }}
+              onEnterKey={loginButtonPress}
+            />
+            <p
+              className="forgot-message"
+              onClick={() => setShowPopUp(true)}
+            >{`Forgot Password?`}</p>
+
+            <div className="login-button-container">
+              <ButtonOutlined
+                label="Create account"
+                isSecondary
+                onClick={() => {
+                  navigate('/sign-up');
+                }}
+              />
+
+              <Button label={'Log in'} onClick={loginButtonPress} />
+            </div>
+            <ErrorSuccessBox content={loginError} error={true} />
+          </div>
+        </div>
+        <div className={`login-loading ${isLoading === true ? 'login-loading-appear' : ''}`}>
+          <LoadingAnimation size={'60px'} />
+        </div>
+
+        <PopupModal
+          trigger={showPopUp}
+          setTrigger={setShowPopUp}
+          blurBackground={false}
+          exitIcon={true}
+          heading={popupTitle}
+          bodyText={popupBodyText}
+        >
+          <ForgotPassword trigger={showPopUp} setTrigger={setShowPopUp} />
+        </PopupModal>
+
+        <LoginBackgroundImages />
+      </div>
+    </>
+  );
+};
+
+const LoginBackgroundImages = () => {
+  return (
+    <>
+      <div className="login-bg-images">
+        <img className="mountain-back" src={MountainB} alt="mountain"></img>
+        <img className="mountain-front-right" src={MountainFR} alt="mountain"></img>
+        <img className="mountain-mid" src={MountainM} alt="mountain"></img>
+        <img className="mountain-front-left" src={MountainFL} alt="mountain"></img>
+
+        <img className="ground" src={Ground} alt="ground"></img>
+        <img className="brachio-left" src={BrachioL} alt="brachiosaurus"></img>
+        <img className="brachio-right" src={BrachioR} alt="brachiosaurus"></img>
+        <img className="ptero" src={Ptero} alt="ptero"></img>
+      </div>
+    </>
+  );
+};
+
+PageLogin.propTypes = {
+  // if username/email/password are invalid
+  incorrectEntry: PropTypes.bool,
+};
+PageLogin.defaultProps = {
+  incorrectEntry: false,
+};
+
+const ForgotPassword = ({ trigger, setTrigger }) => {
+  const [emailError, setEmailError] = useState(''); // error message returned -- either a success or error
+  const [passwordResetSuccess, setpasswordResetSuccess] = useState(false); // success or not
+  const [isLoadingPassword, setIsLoadingPassword] = useState(false);
+  const [emailInput, setEmailInput] = useState(''); // email entered by user
+  const [buttonClick, setButtonClick] = useState(0);
+
+  async function resetButtonPress() {
```

Comment:
```
  async function resetButtonPress() {
```

@Cali0707  this is the function that calls the async resetPassword function and changes states accordingly!